log4j-core-test/src/test/java/org/apache/logging/log4j/core/config/ConfigurationFactoryTest.java

```diff
@@ -130,4 +136,54 @@ void properties(final LoggerContext context) throws IOException {
         final Path logFile = loggingPath.resolve("test-properties.log");
         checkFileLogger(context, logFile);
     }
+
+    @Test
+    void testGetConfigurationWithNullUris() {
+        final ConfigurationFactory factory = ConfigurationFactory.getInstance();
+        final LoggerContext context = new LoggerContext("test");
+
+        assertThrows(NullPointerException.class, () -> factory.getConfiguration(context, "test", (List<URI>) null));
+    }
+
+    @Test
+    void testGetConfigurationWithEmptyUris() {
+        final ConfigurationFactory factory = ConfigurationFactory.getInstance();
+        final LoggerContext context = new LoggerContext("test");
+
+        assertThrows(
+                IllegalArgumentException.class,
+                () -> factory.getConfiguration(context, "test", Collections.emptyList()));
```

Comment:
Could you also add a test for a list containing `null`, please? That is, `Collections.emptyList(null)`.